### PR TITLE
feat(virtual-adoption): basic info

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-259](https://klau.atlassian.net/browse/FSU-259)

## Describe your changes
This PR adds basic info about the virtual adoption process.

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    heading: "Co się dzieje, gdy pies zostanie adoptowany?",
+    icon: "responsibility",
+    description:
+      "Adopcja rzeczywista ma zawsze pierwszeństwo, ale zostaniesz o tym natychmiast poinformowany. Zaproponujemy Ci innego pupila, który jest w potrzebie, abyś mógł kontynuować adopcję.",
+  },
+  {
+    heading: "Czy otrzymuje coś w zamian?",
+    icon: "love",
+    description:
+      "Oczywiście! Otrzymasz swój CERTYFIKAT Wirtualnego Opiekuna! Dodatkowo, otrzymujesz na bieżąco zdjęcia adoptowanego zwierzaka, informacje dotyczące jego stanu zdrowia, kosztów utrzymania oraz filmiki.",
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:36]: "CERTYFIKAT Wirtualnego Opiekuna" should be bolded :D

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                  icon={icon}
+                />
+              </Styled.FeatureItem>
+              <Styled.AdditionalNote variant="bodyTitle">
+                Swoje zwierzę wybrać możesz również poprzez stronę{" "}
+                <a
+                  href="www.ratujemyzwierzaki.pl/sterczaceuszy"
+                  target="_blank"
+                >
+                  www.ratujemyzwierzaki.pl/sterczaceuszy
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:87]: You can use  `{globalData.fundraising.text} ` here 
also I see that this link doesn't work 👀

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                        line={false}
+                      />
+                    }
+                    accountNumber={globalData.accountNumber}
+                  >
+                    <Typography variant="bodyTitle">
+                      Fundacja {globalData.organizationName} <br /> ul.{" "}
+                      {globalData.address.street}
+                      , <br /> {globalData.address.zipCode} <br /> NIP
+                      {globalData.nip} <br />
```
Comment (paulgrym) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:107]: There is double "ul." and the space missing after "NIP"

File: packages/design-system/patterns/panel/Panel.styled.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ export const PanelContent = styled.div`
   padding: 20px;
   border: 1px dashed ${({ theme }) => theme.colors.primary[400]};
   border-radius: 10px;
+  background: ${({ theme }) => theme.colors.neutrals[100]};
```
Comment (paulgrym) [inline packages/design-system/patterns/panel/Panel.styled.js:12]: I know it was another ticket, but in Panel component IconButtons is missing ariaLabel prop

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                      Tytuł przelewu: Wsparcie dla Fundacji
+                    </Typography>
+                  </Panel>
+                </Styled.PanelWrapper>
```
Comment (paulgrym) [inline packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:111]: Icon and title on top of the panel looks weird on mobile 
image

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx (hunk 2 above)
Comment (paulgrym) [reply packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:87]: https:// and rel="noreferrer" missing
`<a
                  href="https://www.ratujemyzwierzaki.pl/sterczaceuszy"
                  target="_blank"
                  rel="noreferrer"
                >`

File: packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx (hunk 5 above)
Comment (im-klau) [reply packages/frontend/app/templates/Adoption/components/VirtualAdoption/VirtualAdoption.jsx:109]: Copy has changed to "Zlecenie stałe" so it looks better now 😅

**Pull request merged**